notebooks/responsible_ai/explainable_ai/solutions/xai_text_with_LIT.ipynb

```diff
@@ -0,0 +1,2770 @@
+{
```

Comment:
<p>I think it's nice that you have outlined all the changes in the markdown above. But because this is a huge code block with a pretty complicated class, can you also add explanations in the comments to reflect the changes you made in the class?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/527/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
<p>That makes sense. I added comment blocks to clarify which part is updated without repeating the explanations in the markdown!</p>